include/seqan3/core/type_traits/basic.hpp

```diff
@@ -15,20 +15,20 @@
 #include <tuple>
 #include <type_traits>
 
-#include <seqan3/core/metafunction/function.hpp>
+#include <seqan3/core/type_traits/function.hpp>
 
 namespace seqan3
 {
 
-/*!\addtogroup metafunction
+/*!\addtogroup type_traits
  * \{
  */
 
 // ----------------------------------------------------------------------------
 // remove_cvref_t
 // ----------------------------------------------------------------------------
 
-/*!\brief Return the input type with `const`, `volatile` and references removed [Type metafunction].
+/*!\brief Return the input type with `const`, `volatile` and references removed (type trait).
```

Comment:
here lower case

Reply:
yes it should be lower case unless it describes a concept. good catch